contributions/essay/fbjorkma-adahen/README.md

```diff
@@ -0,0 +1,9 @@
+## Essay: How to use Jira for agile project managment
+
+### Members:
+Fredrik Björkman (fbjorkma@kth.se) Github: [fbjorkman](https://github.com/fbjorkman)
+
+Adam Henriksson (adahen@kth.se) Github: [Adamih](https://github.com/Adamih) 
+
+### Proposal:
+We want to dive into how Jira works, the advantages you can gain from using Jira in your projects and the value of implementing agile project managment in your company using these kind of tools.
```

Comment:
Hi @fbjorkman, Jira is quite old technology. But it is also very much used in industry. Can you give an outline of the sections of your essay?

Reply:
I've added an outline proposal in the readme